tasks.yaml

```diff
@@ -0,0 +1,58 @@
+includes:
+  - create: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.3/tasks/create.yaml
+  - lint: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.3/tasks/lint.yaml
+  - pull: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.3/tasks/pull.yaml
+  - deploy: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.3/tasks/deploy.yaml
+  - setup: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.3/tasks/setup.yaml
+  - dependencies: ./tasks/dependencies.yaml
+  - test: ./tasks/test.yaml
+
+variables:
+  - name: BUNDLE_PATH
+    default: bundles/k3d/dev
+
+tasks:
+  - name: default
+    description: Create K3D Cluster with istio + SWF
+    actions:
+      - task: create-swf-ci-test-bundle
+      - task: setup:k3d-test-cluster
+      - task: deploy:test-bundle
+        with:
+          path: ${BUNDLE_PATH}
+
+  - name: create-swf-ci-test-bundle
+    description: Create the UDS bundle with Gitlab and its dependencies
+    actions:
+      - task: dependencies:create
+        with:
+          options: "-a ${UDS_ARCH}"
+      - task: create:test-bundle
+        with:
+          path: ${BUNDLE_PATH}
+
+  - name: create-test-bundle
+    description: Create UDS SWF bundle with dependencies
+    actions:
+      - task: create:test-bundle
+        with:
+          path: ${BUNDLE_PATH}
```

Comment:
do we need this?

Reply:
yeah you cant call individual tasks with a with: directly

Reply:
i see what you meant. I actually updated the create-swf-ci-test-bundle to just be create-test-bundle